RaceTrack/PresentationLayer/Program.cs

```diff
@@ -0,0 +1,25 @@
+﻿using TournamentManagement;
+using DataRace;
+using System.Diagnostics;
+using System.Runtime.ConstrainedExecution;
+
+class Program
+{
+    static async Task Main(string[] args)
+    {
+        var tournament = new TournamentResult(new RacerFactory().GetRecers());
+
+        tournament.OnRaceCompleted = (racerModels) =>
+        {
+            Console.WriteLine($"{racerModels.FirstRacer.Name} vs {racerModels.SecondRacer.Name}: Winner is {racerModels.Winner.Name}");
+        };
```

Comment:
Better make a static method in Main and pass this method reference here. will look cleaner